alchemy/src/cloudflare/inherit.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Cloudflare Workers binding for inherting existing bindings.
+ *
+ * If a binding exists on another worker it can be copied to a new worker deployment.
+ *
+ * @example
+ * ```ts
+ * import { Worker, Inherit } from "alchemy/cloudflare";
+ *
+ * await Worker("my-worker", {
+ *   name: "my-worker",
+ *   entrypoint: "./src/worker.ts",
+ *   bindings: {
+ *     KEY: Inherit("Key")
+ *   }
+ * });
+ * ```
+ */
+export function Inherit(options?: { oldName?: string; versionId?: string }) {
```

Comment:
Is there a reason we didn't return type `Inherit`? Wondering if this might mess up syntax highlighting